rules.md

```diff
@@ -555,45 +553,35 @@ This section relates to the special Admin role and rules on how they operate.
 
 Admins work to facilitate the production of output and regulate the operations of the fund itself.
 
-Admins earn rewards in the form of a % commission added to the value of the Tasks on Bounties that they manage, as defined in the [Price List](#25-price-list). 
-
-This is to incentivize ownership of the Tasks in terms of defining them, promoting them and supporting the work involved in their completion.  
-
-An Admins role is reputation based on the history of Bounties they have delivered, and as everything in the App is public, all metrics for an Admins performance can be mined from the App’s data.
+Admins earn commission on Bounty Tasks where they function as Primary Admin, as defined in the [Price List](#25-price-list).  This is to incentivize ownership of the Tasks in terms of defining them, promoting them and supporting the work involved in their completion.
 
-The number of users given Admin rights is limited and Admins effectively compete with each other on performance (how good is their output). 
+An Admin's reputation is based on the history of Bounties they have delivered, and as everything in the App is public, all metrics for an Admins performance can be mined from the App’s data.
 
-The least performing Admin roles may be replaced with new Admins periodically at the discretion of the Proposal owner.
+The number of users given Admin rights is limited and Admins effectively compete with each other on performance (how good is their output). Low performing Admins may be replaced with new Admins periodically at the discretion of the Proposal owner.
 
-Ultimately the Admins main goal is to get as many bounties completed as quickly as possible, so that they can maximize commission and secondly grow a list of successful and popular bounties that deliver the most value at the least cost when compared to other Admins in the system.
+The Admins main goal is to get as many bounties completed as quickly as possible, so that they can maximize commission and secondly grow a list of successful and popular bounties that deliver the most value at the least cost when compared to other Admins in the system.
```

Comment:
The [Primary] Admin's or The Admins' (plural) main goal? It is not clear how this is the goal of the secondary admin, since they probably don't take any role in directing the project.

Reply:
I was trying to limit this commit to substance changes rather than grammar, but this one snuck through, which kind of opens a small can of worms.  I somewhat disagree with this paragraph in general, but I didn't mean to edit it in this PR because it would be mixing issues.  So for the purpose of this PR (simply adding the secondary admin concept) I would say lets change it to read "An admin's main goal...".  There are many instances in the document where I don't explicitly  distinguish between primary and secondary admin, because I consider them two different responsibilities with respect to a given bounty, but still one singular role; their overall purpose and goal should be the same.

Reply:
Yes, I agree, this is probably a matter for a future PR addressing some of the existing grammatical & other idiosyncrasies, rather than the subsatnce of your changes. We can refer back to it when addressing those at a later date.